compiler/rustc_codegen_llvm/src/gotoc/utils.rs

```diff
@@ -25,6 +25,14 @@ impl<'tcx> GotocCtx<'tcx> {
         }
     }
 
+    // Special naming conventions for parameters that are spread from a tuple
+    // into its indivdual components at the LLVM level, see comment at
+    // compiler/rustc_codegen_llvm/src/gotoc/mod.rs:codegen_function_prelude
+    pub fn codegen_spread_arg_name(&self, l: &Local) -> String {
+        let fname = self.current_fn().name();
+        format!("{}::1::spread{:?}", fname, l)
```

Comment:
Do we need to hard-code this?

Reply:
I think so, the name `spread` is not special, we just need to name it something different from `var` and I chose this to try and help convey the purpose.